docs/src/CommutativeAlgebra/ca.md

```diff
@@ -16,8 +16,21 @@ Pages = ["integer.md"]
 
 This part of OSCAR is under development with regard to providing both the functionality and the documentation. At current state, the documentation serves as a guide on what to do and will be constantly updated.
 
-## Ideal constructors
+## Ideal Constructors for Multivariate Polynomial Rings
 
+```@docs
+ideal(g::Array{T, 1}) where {T <: MPolyElem}
+```
+```@docs
+ideal(Rx::MPolyRing, g::Array{<:Any, 1})
+```
+###### Example
+
+```@repl oscar
+R, (x, y) = PolynomialRing(QQ, ["x", "y"])
+I = ideal([x*y-3*x,y^3-2*x^2*y])
+J = ideal(R, [(x+y)^2])
+```
```

Comment:
As mentioned else where, I would suggest moving these examples into the docstrings of the relevant methods instead, so that they also show up in the REPL help. But this can of course be done in a future PR.